maxtext_dependencies.Dockerfile

```diff
@@ -4,6 +4,9 @@ FROM python:3.10-slim
 # Install system dependencies and Git
 RUN apt-get update && apt-get install -y curl gnupg git
 
+# Install iproute2 ethtool lsof
+RUN apt-get update && apt-get install -y iproute2 ethtool lsof
```

Comment:
Shoudl we add these to line #5? Just thinking out loud, not sure which is better or clearer.

Reply:
Yeah it could be added to that line but I think it is clearer to keep it separate as the dependencies for adjusting network?

Reply:
Probably should have the unifying comment then. But nbd this looks pretty good.